config/test.exs

```diff
@@ -23,6 +23,7 @@ if File.exists?(data_path) do
 end
 
 config :livebook, :data_path, data_path
+config :livebook, :feature_flags, deployment_groups: true
```

Comment:
```suggestion
```

We enable all feature flags except prod in main. :)

Reply:
@josevalim actually I reverted to the same approach as here, because `config :livebook, :feature_flags, []` in prod merges the flags rather than overriding. If there's a better way let me know!

Reply:
I see. You have already added the flag anyway, so we can remove this still. :)

Reply:
Yup!